addons/product/models/product_template.py

```diff
@@ -69,11 +69,9 @@ def _read_group_categ_id(self, categories, domain, order):
     list_price = fields.Float(
         'Sales Price', default=1.0,
         digits='Product Price',
-        help="Price at which the product is sold to customers.")
-    # lst_price: catalog price for template, but including extra for variants
-    lst_price = fields.Float(
-        'Public Price', related='list_price', readonly=False,
-        digits='Product Price')
+        help="Price at which the product is sold to customers.",
+        help_links=['http://google.com'],
```

Comment:
:thinking:

Reply:
Just a relic that should be in here https://runbot.odoo.com/runbot/bundle/master-link-error-tooltip-wan-90511